compsets/all.input

```diff
@@ -23,8 +23,8 @@ run cpld_fv3_ccpp_384_mom6_cice_1d_bmark_rt @ fv3, coupledapp, warm384, physics,
 run cpld_fv3_ccpp_384_mom6_cice_ww3_cold_bmark_rt @ fv3, coupledapp, coldBM, physics, bm, ww3
 run cpld_fv3_ccpp_384_mom6_cice_ww3_1d_bmark_rt @ fv3, coupledapp, warm384, physics, bm, ww3
 
-run cpld_fv3_ccpp_mom6_cice_cold_debug @ fv3, coupledapp, cold, physics, debug
-run cpld_fv3_ccpp_mom6_cice_6h_debug @ fv3, coupledapp, warm, physics, debug
+#run cpld_fv3_ccpp_mom6_cice_cold_debug @ fv3, coupledapp, cold, physics, debug
```

Comment:
May I ask why we comment out these two debug tests?

Reply:
Debug tests are not implemented in rt.sh yet. Since the new baselines (20200504) were generated using rt.sh, I had to comment those lines.